internal/pkg/term/selector/selector.go

```diff
@@ -62,14 +64,15 @@ For example: 0 17 ? * MON-FRI (5 pm on weekdays)
 
 // Final messages displayed after prompting.
 const (
-	appNameFinalMessage = "Application:"
-	envNameFinalMessage = "Environment:"
-	svcNameFinalMsg     = "Service name:"
-	jobNameFinalMsg     = "Job name:"
-	deployedSvcFinalMsg = "Service:"
-	taskFinalMsg        = "Task:"
-	workloadFinalMsg    = "Name:"
-	dockerfileFinalMsg  = "Dockerfile:"
+	appNameFinalMessage  = "Application:"
+	envNameFinalMessage  = "Environment:"
+	svcNameFinalMsg      = "Service name:"
+	jobNameFinalMsg      = "Job name:"
+	deployedSvcFinalMsg  = "Service:"
+	taskFinalMsg         = "Task:"
+	workloadFinalMsg     = "Name:"
+	dockerfileFinalMsg   = "Dockerfile:"
+	fmtTopicFinalmessage = "Environment %s subscriptions:"
```

Comment:
nit: Should we say just: 
```suggestion
	fmtTopicFinalmessage = "Subscriptions:"
```
Since they can only deploy to one environment at a time, I dunno if we need to show the env name again

Reply:
This is used in `svc init` to select topics from one environment at a time; we need to use the results here to generate YAML for the manifest like:
```
Which topics would you like to subscribe to in the `test` environment?
[x] orders (api)
[ ] signups (users)

Which topics would you like to subscribe to in the `prod` environment?
[x] orders (api)
[x] signups (users)
```

```yaml
// Where the customer selects `orders (api)` for both `test` and `prod` environments
subscribe:
  topics:
    - service: api
      name: orders

// Where the customer selects `signups (users)` only in the `prod` environment. 
environments:
  prod:
    subscribe:
      topics:
        - service: users
          name: signups
```

Reply:
ooh I seee 🤔 that makes sense! We can keep it as is then

Most likely they will sign up to the same set of topics across environments, this is definitely tricky 💭 I'd like to reduce if possible the number of prompts to the customers to just fill enough info in the manifest and then let them write it.

What do you think of fetching the `(topic, service)` pairs across environments and only prompting for the ones that are shared?
```
Which common topics would you like to subscribe to?
- [ ] orders (api)
- [ ] signups (users)
```
And then we don't prompt beyond that, they can manually edit the manifest if they want different topics at this point per env.

Reply:
maybe I am overthinking this! it might be okay to prompt a few more times :shipit: